### ISSUE TITLE
ctrl-z bg doesn't work

### ISSUE THREAD

**Issue by davidbrochart** [scopatz/xonsh#406]
Pressing ctrl-z suspends the current job, but then entering `bg` doesn't make that job run in the background. It remains stopped although `jobs` says it's running. After entering the following code, `gedit` is frozen.

```
david@david-VirtualBox ~ $ gedit README 
^Z
*[1] stopped: gedit README & (5427)
david@david-VirtualBox ~ $ jobs 
*[1] stopped: gedit README & (5427)
david@david-VirtualBox ~ $ bg 
*[1] running: gedit README & (5427)
david@david-VirtualBox ~ $ jobs 
*[1] running: gedit README & (5427)
david@david-VirtualBox ~ $  
```

**Comment by scopatz** [comment]
Thanks for reporting this @davidbrochart. I'd appreciate any help that I can get here.  The code should mostly be in jobs.py.  Also, can you please tell us a bit more about your system, please? Thanks.

**Comment by adqm** [comment]
This may have come out of the "fix" for vim/emacs not backgrounding properly before in #309 .  I have seen this with evince before, and rolling back fixed.  Work is a bit hectic now, but I can try to find some time to work on this.

**Comment by scopatz** [comment]
Thanks a million @adqm!

**Comment by willsALMANJ** [comment]
This seems to be fixed for me now (possibly fixed by #867 and #962?)

**Comment by scopatz** [comment]
Great! I will go ahead and close this, then, but please feel free to reopen!
